api/src/main/java/org/openmrs/module/openconceptlab/OpenConceptLabActivator.java

```diff
@@ -41,10 +50,49 @@ public void contextRefreshed() {
 		if (!Context.isSessionOpen()) {
 			Context.openSession();
 		}
-			
+
+		String loadAtStartupPath = Context.getAdministrationService()
+				.getGlobalProperty(OpenConceptLabConstants.GP_OCL_LOAD_AT_STARTUP_PATH);
+		if (StringUtils.isBlank(loadAtStartupPath)) {
+			loadAtStartupPath =
+					new File(new File(new File(OpenmrsUtil.getApplicationDataDirectory(), "ocl"), "configuration"), "loadAtStartup").getAbsolutePath();
+			Context.getAdministrationService()
+					.saveGlobalProperty(
+							new GlobalProperty(OpenConceptLabConstants.GP_OCL_LOAD_AT_STARTUP_PATH, loadAtStartupPath)
+					);
+		}
+
+		File loadAtStartupDir = new File(loadAtStartupPath);
+		if (!loadAtStartupDir.exists()) {
+			loadAtStartupDir.mkdirs();
+		}
+
 		UpdateScheduler scheduler = Context.getRegisteredComponent("openconceptlab.updateScheduler", UpdateScheduler.class);
-		scheduler .scheduleUpdate();
-		
+
+		File[] files = loadAtStartupDir.listFiles();
+		if (files != null && files.length > 1) {
+			throw new IllegalStateException(
+					"There is more than one file in ocl/loadAtStartup directory\n" +
+					"Ensure that there is only one file\n" +
+					"Absolute directory path: " + loadAtStartupDir.getAbsolutePath());
+		} else if (files != null && files.length != 0) {
+			File file = files[0];
+			if (file.getName().endsWith(".zip")) {
+				InputStream inputStream;
+				try {
+					inputStream = Utils.extractExportInputStreamFromZip(new ZipFile(file));
+					Importer importer = Context.getRegisteredComponent("openconceptlab.importer", Importer.class);
+					importer.setInputStream(inputStream);
+					scheduler.scheduleNow();
+					file.delete();
```

Comment:
You don't copy the inputStream... yet you delete the file... How is the importer going to read the file then?